CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v1.2.0] 2020-01-13
+
+### Changed
+
+- Upgrade to nginx-ingress-controller 0.27.0.
+
```

Comment:
Text is good but our changelogs are still quite inconsistent.  In some apps we're linking to the PR. Should we do that here? 

e.g. [v1.1.3]: https://github.com/giantswarm/coredns-app/pull/11

Also should we link to the upstream release notes?

Reply:
There can be more PRs/changes in single app release, and there will be more in this one, to be added with additional PRs. Just added a link to this PR at the end of the initial changelog entry. PTAL

Reply:
Yes good point about multiple PRs. This LGTM and mainly I'd just like our changelogs to be more consistent. 

As we're now including the link to our changelog in the changelogs exposed by cluster-operator.

Reply:
I just made versions in changelog link to GitHub releases; does that make sense?

Reply:
Yes good idea.